test/co-reputation/get_reputation.js

```diff
@@ -23,17 +23,23 @@ contract('Coreputation::getReputation', (accounts) => {
   let accountProvider;
   let coReputation;
   let validatorInfo;
+  let coconsensus;
 
   beforeEach(async () => {
     accountProvider = new AccountProvider(accounts);
     coReputation = await CoReputation.new();
```

Comment:
`Coreputation`

Reply:
also the test directory should not be "co-reputation" but "coreputation"

Reply:
Done!